### ISSUE TITLE
Accessibility issues with GiveWP forms

### ISSUE THREAD

**Issue by mathetos** [impress-org/givewp#4697]
# Bug Report
As reported from a customer:

> 2 Accessibility Breaks:
> 
> 1) On the modal after the customer hits "Donate":
> The "X" in the top right corner used to back out of the form, is not properly titled as "cancel" or whatever makes sense to you. As a result screen readers, while they can "see" it, can not tell what it is or what it does. Please fix.
> 
> 2) The order of fields in the default form, namely the custom input field appearing before the suggested donation amount buttons, is confusing to users with screen readers as well as those with general cognitive issues. The suggested amount buttons should be first, with the custom amount field below. If possible, please fix.
> 
> Thanks so much! :D
> 

Reported here:
https://secure.helpscout.net/conversation/1152427051/64305?folderId=672194
